Content/LC-switch/lc_switch.js

```diff
@@ -119,10 +119,10 @@
 	
 	
 	// handlers
-	$(document).ready(function() {

```

Comment:
Na ez az LC-switch még egy viszonylag karbantartott library: https://github.com/LCweb-ita/LC-switch
És ha jól látom, akkor tavaly januárban volt is egy release, ami első ránézésre jQuery 3.6 kompatibilis (nyilván, hiszen átírták vanilla javaScriptre).
Meg tudod azt tenni, hogy kicseréled ezt az LC Switchet a legújabb verzióval? Itt a release, az ebben lévő fájlokat kellene bepakolni ebbe a mappába: https://github.com/LCweb-ita/LC-switch/releases/tag/v2.0.3
Bower, preview.png, package.json nem kell.

Ha jól látom, akkor a stylingot is a js-sel húzza be a szerző (mert meh), így az lc_switch.css nem fog kelleni, ami azt jelenti, hogy a hozzá tartozó style-t a resource manifestből lehet törölni a constanssal együtt.

Tesztelni tudod mondjuk a Patient Link dashboardján az Expand All kapcsolóval.

Nagyon fasza lenne, ha működne, de ha nem, akkor hagyd és maradjon így. Csak ekkor ebben az esetben itt is legyen szintén egy Mods.txt, ahova beírjuk, hogy fel lett húzva a cucc 3.6-ra.

Reply:
Megcsináltam. Ahol használva volt, átírtam az inicializálást az újra. Nekem jónak tűnik, de nem tudom hogy például minden class meg ilyenek maradt-e a régi. (De amit néztem ott okés.)